packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -84,8 +84,13 @@ class MinimumTapTargetGuideline extends AccessibilityGuideline {
         result += traverse(child);
         return true;
       });
+      if (node.isMergedIntoParent)
```

Comment:
Can you add tests to flutter_test that exercise all these edge cases you're fixing here to ensure they continue to work in the future?

Reply:
I expanded the test coverage and fixed similar issues to get text contrast working ... there were a lot of bugs in my implementation :(